documentation/src/main/asciidoc/topics/proc_using_jgroups_default.adoc

```diff
@@ -1,45 +1,41 @@
-[id='jgroups_getting_started-{context}']
-= Getting Started with Default Stacks
+[id='using-jgroups-default-stacks_{context}']
+= Using the Default JGroups Stacks
 {brandname} uses JGroups protocol stacks so nodes can send each other messages on dedicated cluster channels.
 
 {brandname} provides preconfigured JGroups stacks for `UDP` and `TCP` protocols.
 You can use these default stacks as a starting point for building custom cluster transport configuration that is optimized for your network requirements.
 
 .Procedure
 
-. Locate the default JGroups stacks, `default-jgroups-*.xml`, in the `default-configs` directory inside the `infinispan-core-{ispn_version}.jar` file.
-ifdef::server_mode[]
-+
-The `jar` file is in the `{server_home}/lib` directory.
-+
-endif::server_mode[]
-. Do one of the following:
+Do one of the following to use one of the default JGroups stacks:
 
-* Use the `stack` attribute in your `infinispan.xml` file.
+* Declare the `stack` attribute in your `infinispan.xml` file.
```

Comment:
I believe "Use" was better. In XML "declaration" means something else: https://www.tutorialspoint.com/xml/xml_declaration.htm

Reply:
Changed back to "Use". Thanks @danberindei